lib/java_buildpack/framework/luna_security_provider.rb

```diff
@@ -144,20 +148,27 @@ def write_configuration(servers, groups)
 VirtualToken = {
 EOS
           groups.each_with_index { |group, index| write_group f, index, group }
-          write_epilogue f
+          write_epilogue f, groups
         end
       end
 
-      def write_epilogue(f)
+      def write_epilogue(f, groups)
         f.write <<EOS
 }
 
 HAConfiguration = {
   AutoReconnectInterval = 60;
   HAOnly = 1;
-  ReconnAtt = 20;
+  reconnAtt = -1;
```

Comment:
Where this is problematic is if the configuration points to something that will never exist.  It means that it will retry forever (also, not sure what the backoff strategy is) without ever failing.  We generally consider it a bad practice to retry without limit.  But if you all say it's good, I'll let it stand.